src/System.CommandLine/Builder/CommandLineBuilderExtensions.cs

```diff
@@ -475,19 +436,9 @@ public static CommandLineBuilder UseHelp(this CommandLineBuilder builder, int? m
         /// <returns>The same instance of <see cref="CommandLineBuilder"/>.</returns>
         public static CommandLineBuilder UseParseDirective(
             this CommandLineBuilder builder,
-            int? errorExitCode = null)
+            int errorExitCode = 1)
```

Comment:
IMO it's better to make the default value explicit. It comes from 

https://github.com/dotnet/command-line-api/blob/7030070c394c2d85dd10f6d1cc22ac6c1e902ce2/src/System.CommandLine/Invocation/ParseDirectiveResult.cs#L17